<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LitoSoler" name="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades" tests="15" failures="0" timestamp="14 Jan 2021 15:47:48 GMT" time="653.886" errors="2">
  <testcase name="InformacaoBlocos" time="18.692" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="criarBlocos" time="23.844" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="executarBlocos" time="132.466" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="visualizarAN3" time="49.286" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="mdrEntrar" time="27.965" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="criar" time="122.660" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="editarBlocos" time="20.877" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="informacao" time="30.963" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="visualizarActualizacao" time="11.732" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades">
    <error type="java.lang.AssertionError" message="As informações não são as esperadas expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: As informações não são as esperadas expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.timp.test.DFG.SPED.ECF.ECFFuncionalidades.visualizarActualizacao(ECFFuncionalidades.java:166)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- visualizarActualizacao -->
  <testcase name="visualizar" time="24.139" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="visualizarBlocos" time="19.111" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="ID" time="75.561" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="ingresar" time="7.813" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
  <testcase name="editar" time="41.702" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades">
    <error type="java.lang.AssertionError" message="O valor do campo não foi editado expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: O valor do campo não foi editado expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.timp.test.DFG.SPED.ECF.ECFFuncionalidades.editar(ECFFuncionalidades.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- editar -->
  <testcase name="Gravar" time="47.075" classname="com.timp.test.DFG.SPED.ECF.ECFFuncionalidades"/>
</testsuite> <!-- com.timp.test.DFG.SPED.ECF.ECFFuncionalidades -->
